roles/jd-client/src/lib/job_declarator/mod.rs

```diff
@@ -114,7 +114,7 @@ impl JobDeclarator {
             allocated_tokens: vec![],
             req_ids: Id::new(),
             min_extranonce_size,
-            last_declare_mining_job_sent: None,
+            last_declare_mining_jobs_sent: HashMap::with_capacity(10),
```

Comment:
why initialize via `with_capacity(10)`? seems quite arbitrary?
why not simply `new()`?

Reply:
I put 10 in order to not leak memory

Reply:
see my comment above, initializing it with 10 do not do anything in order to not leak memory